db/migrations/initial_schema.js

```diff
@@ -11,7 +11,8 @@ exports.up = knex => {
     })
     .createTable("breaches", table => {
       table.increments("id").primary();
-      table.string("name").unique();
+      table.string("name");
+      table.string("domain");
```

Comment:
I don't know enough about Knex migrations to know if we can add columns here, or if it needs to be in some migration file.

Reply:
I had to do the psql dropdb/createdb song and dance and then it worked.

Reply:
I'm also fine w/ ignoring this and submitting a follow up issue.

Reply:
I'm fine with this (i.e. dropping the db and resetting) for now since we don't have any data we care about yet

Reply:
> Sure, works for me, as long as it will work in stage/prod without needing too much OPs work.

That's fine for now.  In future we won't want to do anything that way, and also we are getting pretty close to the wire so please let me know when there are configuration and ops related tasks to run any given version of the code we want released.